src/modeling_library/distributions/normal.jl

```diff
@@ -85,10 +83,29 @@ function logpdf_grad(::BroadcastedNormal,
     assert_has_shape(x, broadcast_shapes_or_crash(mu, std);
                      msg="Shape of `x` does not agree with the sample space")
     z = (x .- mu) ./ std
-    deriv_x = sum(- z ./ std)
+    deriv_x = - z ./ std
     deriv_mu = -deriv_x
-    deriv_std = sum(-1. ./ std .+ abs2.(z) ./ std)
-    (deriv_x, deriv_mu, deriv_std)
+    deriv_std = -1. ./ std .+ abs2(z) ./ std
```

Comment:
We need to keep `z`'s shape, right?
```suggestion
    deriv_std = -1. ./ std .+ abs2.(z) ./ std
```

Reply:
Yeah, you are absolutely right!